sandbox/public/index.html

```diff
@@ -71,7 +71,8 @@
           }
         },
         data: {
-          nonXdmKey: "nonXdmValue"
+          nonXdmKey: "nonXdmValue",
+          name: "home"
```

Comment:
It might be good to add a comment here as to the importance of `name`.

Reply:
Jinx!